src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -49,21 +48,20 @@ public void SetUp()
             routingTopology = new ConventionalRoutingTopology(true);
             receivedMessages = new BlockingCollection<IncomingMessage>();
 
-            var settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
+            var endpointName = "NServiceBus.Routing.EndpointName";
```

Comment:
To match previous behavior, this should be set to "endpoint" instead.

However, I think an even better option would be to remove `endpointName` completely and pass `ReceiverQueue` directly into the parser instead. That actually matches the real behavior better anyway.